src/pytestqt/qtbot.py

```diff
@@ -59,30 +59,30 @@ class QtBot:
 
     Below are methods used to simulate sending key events to widgets:
 
-    .. staticmethod:: keyClick (widget, key[, modifier=Qt.NoModifier[, delay=-1]])
-    .. staticmethod:: keyClicks (widget, key_sequence[, modifier=Qt.NoModifier[, delay=-1]])
-    .. staticmethod:: keyEvent (action, widget, key[, modifier=Qt.NoModifier[, delay=-1]])
-    .. staticmethod:: keyPress (widget, key[, modifier=Qt.NoModifier[, delay=-1]])
-    .. staticmethod:: keyRelease (widget, key[, modifier=Qt.NoModifier[, delay=-1]])
+    .. staticmethod:: keyClick (widget, key[, modifier=Qt.KeyboardModifier.NoModifier[, delay=-1]])
+    .. staticmethod:: keyClicks (widget, key_sequence[, modifier=Qt.KeyboardModifier.NoModifier[, delay=-1]])
+    .. staticmethod:: keyEvent (action, widget, key[, modifier=Qt.KeyboardModifier.NoModifier[, delay=-1]])
+    .. staticmethod:: keyPress (widget, key[, modifier=Qt.KeyboardModifier.NoModifier[, delay=-1]])
+    .. staticmethod:: keyRelease (widget, key[, modifier=Qt.KeyboardModifier.NoModifier[, delay=-1]])
 
         Sends one or more keyboard events to a widget.
 
         :param QWidget widget: the widget that will receive the event
 
-        :param str|int key: key to send, it can be either a Qt.Key_* constant or a single character string.
+        :param str|int key: key to send, it can be either a Qt.Key.Key_* constant or a single character string.
 
         .. _keyboard modifiers:
 
         :param Qt.KeyboardModifier modifier: flags OR'ed together representing other modifier keys
             also pressed. Possible flags are:
 
-            * ``Qt.NoModifier``: No modifier key is pressed.
-            * ``Qt.ShiftModifier``: A Shift key on the keyboard is pressed.
-            * ``Qt.ControlModifier``: A Ctrl key on the keyboard is pressed.
-            * ``Qt.AltModifier``: An Alt key on the keyboard is pressed.
-            * ``Qt.MetaModifier``: A Meta key on the keyboard is pressed.
-            * ``Qt.KeypadModifier``: A keypad button is pressed.
-            * ``Qt.GroupSwitchModifier``: X11 only. A Mode_switch key on the keyboard is pressed.
+            * ``Qt.KeyboardModifier.NoModifier``: No modifier key is pressed.
```

Comment:
To clarify: is this change only for PySide 2?

Reply:
This change is only required for PyQt6:

```
In [9]: from PyQt6.QtCore import Qt

In [10]: Qt.NoModifier
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[10], line 1
----> 1 Qt.NoModifier

AttributeError: type object 'Qt' has no attribute 'NoModifier'

In [11]: Qt.KeyboardModifier.NoModifier
Out[11]: <KeyboardModifier.NoModifier: 0>
```
In all other binding both versions exist. See, e.g., https://www.pythonguis.com/faq/pyqt5-vs-pyqt6/

Reply:
Awesome, thanks!